components/availability/src/Availability.svelte

```diff
@@ -1145,6 +1182,44 @@
           }
         }
       }
+
+      .slot-list {
+        display: grid;
+        grid-auto-rows: 50px;
+        grid-auto-flow: row;
+        overflow: auto;
+        gap: 0.25rem;
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+
+        .slot {
+          border-radius: 4px;
+          background: transparent;
+          position: relative;
+          align-items: center;
+          justify-content: center;
+          align-content: center;
+          font-family: sans-serif;
+          border: 1px solid rgba(0, 0, 0, 0.1);
+          box-shadow: none !important;
+
+          &:hover {
+            box-shadow: none;
+            background-color: rgba(0, 0, 0, 0.1);
+          }
+
+          &.selected {
+            background: purple;
+            color: white;
+          }
+
+          .partial {
+            display: block;
+            font-size: 0.6rem;
```

Comment:
what is this, a font-size for ants?! 😝

Reply:
It's tiny but I think any bigger and I'd opt to just not have it -- it really muddies up the visuals!